spec/requests/commodity_spec.rb

```diff
@@ -66,4 +66,36 @@
       end
     end
   end
+
+  context 'when commodity with whatever' do
+    before do
+      VCR.use_cassette('headings#show_0101') do
```

Comment:
Our VCR cassettes are a total mess. So we've historically labelled the commodity response cassette for commodity 0101300000 `headings_show_0101`. At some point I'd love to relabel all of our cassettes based on the feature that needs them or something.

Reply:
That's another large chunk of work... Not today, maybe? 😄